proof/archive.go

```diff
@@ -74,8 +75,16 @@ func (l *Locator) Hash() [32]byte {
 	}
 	buf.Write(l.ScriptKey.SerializeCompressed())
 
+	if l.OutPoint != nil {
+		err := lnwire.WriteOutPoint(&buf, *l.OutPoint)
```

Comment:
Ah, forgot about the error return type. Since we ignore any errors returned by `buf.Write()` above (which AFAIK can never return an actual error on a `bytes.Buffer` but is part of the `io.Writer` interface anyway), we probably can do the same here as well.

Reply:
The was merged before I saw your comment! Let me know if you want me to open another PR to go back on the err handling (I'm guessing this is fine as is?).